### PR TITLE
add accelerate network for update

### PR DESCRIPTION
**Pull request by ChenTanyi**

### CODE REVIEW

File: azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java [hunk 1]
```diff
@@ -518,11 +518,22 @@
...
         interface WithAcceleratedNetworking {
+            /**
+             * Enables accelerated networking.
+             * <p>
+             * Note that additional steps need to be taken in the virtual machine itself for the virtual machine associated with this network interface to be able to
+             * take advantage of accelerated networking. This feature might not be available in some regions, virtual machine sizes, or operating system versions.
+             * It can be enabled only during the creation of a network interface, not during an update.
+             * @return the next stage of the definition
+             */
+            DefinitionStages.WithCreate withAcceleratedNetworking();
```
Comment (xccc-msft) [inline azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java:529]: should return `Update`?

Comment (ChenTanyi) [reply azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/NetworkInterface.java]: done

**Comment by ChenTanyi** [comment]
After testing, if the interface has been attached to VM, it could not be updated to enabled accelerated network.

**Review by weidongxu-microsoft — APPROVED** [review]
From doc it seems that accelerated networking can be changed if VM is deallocated.

**Review by xccc-msft — APPROVED** [review]

**Comment by ChenTanyi** [comment]
@weidongxu-microsoft It seems the travis will always not run

**Pull request merged**